packages/core/src/ceramic.ts

```diff
@@ -58,7 +58,7 @@ const DEFAULT_ANCHOR_SERVICE_URLS = {
   [Networks.MAINNET]: 'https://cas.3boxlabs.com',
   [Networks.ELP]: 'https://cas.3boxlabs.com',
   [Networks.TESTNET_CLAY]: 'https://cas-clay.3boxlabs.com',
-  [Networks.DEV_UNSTABLE]: 'https://cas-dev.3boxlabs.com',
+  [Networks.DEV_UNSTABLE]: 'https://cas-qa.3boxlabs.com',
```

Comment:
Should this be injected from Terraform instead?

Reply:
Ok to do it this way for now, though 👍🏼

Reply:
not really. these should be hardcoded. shouldn't rely on ppl running ceramic daemon to import these via environment variable

Reply:
Ok, makes sense 👍🏼